src/fetch-map/utils.ts

```diff
@@ -67,3 +67,21 @@ export function isRemoteCalculationSupported(dataset: Dataset) {
 
   return true;
 }
+
+const DATE_FORMATTER = new Intl.DateTimeFormat('en-US', {
+  year: '2-digit',
+  month: 'numeric',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric',
+  timeZone: 'UTC',
+});
```

Comment:
I won't worry too much right now, but I'm modifying how dates are being rendered in `cloud-native` through the new `DateRangePicker` component. Under the hood, we use `dayjs` for doing such (which doesn't mean we should be using it here), but I think at some point it could be interesting to use a similar format everywhere, or maybe let the client inject their own format while using `carto-api-client` 👍

Reply:
Nice! Yes `dayjs` looks much more compact, I'd be OK with adding it here. In general I think it's good if we can keep the API client mostly light and unopinionated about views/formatting/UI ... but that goal does conflict a bit with the "reproduce a Builder map" purpose here, so I guess we'll need to work out a compromise at some point. :)